src/Data/survey.js

```diff
@@ -1,30 +1,181 @@
-// import {
-//   car,
-//   train,
-//   walking_biking,
-//   meateater,
-//   vegetarian,
-//   vegan,
-// } from "./intensityKilo";
+import car from "../Shared/Icons/car.png";
+import clothing from "../Shared/Icons/clothing.png";
+import electronics from "../Shared/Icons/electronics.png";
+import house from "../Shared/Icons/house.png";
```

Comment:
You have to update picture names so they start with capital letters. I had to do this manually in order to start the app

Reply:
I suspect this is a bug in Git.
We had a different convention in the team as to file naming. Settled on all lower case and correctly edited both file names and imports. Some of us had to manually correct the files on our local version, even if the correct code was pushed and pulled. I think Git may not check capital letters of .png files when looking to see what to update on local during a pull.
Only conclusion I can think of given what we experienced while sitting next to each other physically.

Not sure what we can do about it.